package.json

```
@@ -34,9 +34,13 @@
     "compile:types": "tsc --emitDeclarationOnly --declaration",
     "compile:browser": "webpack --progress --env target=web",
     "docs": "rimraf docs && typedoc",
-    "test": "npm run test:node && npm run test:browser",
-    "test:browser": "npm run compile:browser && jest --verbose --config=jest.config.ts --selectProjects=dom",
-    "test:node": "jest --verbose --config=jest.config.ts --selectProjects=node",
+    "test": "npm run compile:browser && jest --verbose --config=jest.config.ts --selectProjects=node:unit node:integration dom:integration",
+    "test:integration:browser": "npm run compile:browser && jest --verbose --config=jest.config.ts --selectProjects=dom:integration",
+    "test:integration:node": "jest --verbose --config=jest.config.ts --selectProjects=node:integration",
+    "test:integration": "npm run compile:browser && jest --verbose --config=jest.config.ts --selectProjects=node:integration dom:integration",
```

Later we would change one of the flag of browser or node integration whithin its defined script, but we may forget anytime to do that here. this suggestion sort out this problem. 

```suggestion
    "test:integration": "npm run compile:browser && npm run test:integration:browser && npm run test:integration:node",
```

---

I am against this. See my [comment](https://github.com/ethersphere/bee-js/pull/258#discussion_r612974394) below.